cuegui/cuegui/AbstractGraphWidget.py

```diff
@@ -0,0 +1,187 @@
+from PySide2 import QtCore
+from PySide2 import QtGui
+from PySide2 import QtWidgets
+
+from NodeGraphQt import NodeGraph
+from cuegui.CueNodeGraphQt import CueLayerNode
+
+
```

Comment:
new files should also keep to the convention of including a one line summary description, example `AbstractTreeWidget.py`  to the effect of `Base class for CueGUI ....`